explain-overview.md

```diff
@@ -57,6 +57,79 @@ The following describes the output of the `EXPLAIN` statement above:
 * `access object` shows the table, partition and index that is being accessed. The parts of the index are also shown, as in the case above that the column `a` from the index was used. This can be useful in cases where you have composite indexes.
 * `operator info` shows additional details about the access. See [Operator info overview](#operator-info-overview) for additional details.
 
+> **Note:**
+>
+> In the execution plan, for all Probe side operators of `IndexJoin` and `Apply`, the meaning of `estRows` from v6.4.0 is different that before v6.4.0.
+> Before v6.4.0, `estRows` means the estimated row count expected to be processed by the Probe side operators for each row from the Build side operator. From v6.4.0, `estRows` means the estimated **total row count** expected to be processed by the Probe side operators. The actual row count displayed (the `actRows` column) in the `EXPLAIN ANALYZE` statement means the total row count, so the meanings of `estRows` and `actRows` for those operatos are consistent from v6.4.0.
```

Comment:
```suggestion
> Before v6.4.0, `estRows` means the number of estimated rows to be processed by the probe side operators for each row from the build side operators. Since v6.4.0, `estRows` means the **total number** of estimated rows to be processed by the probe side operators. The actual number of rows displayed (indicated by the `actRows` column) in the result of `EXPLAIN ANALYZE` means the total row count, so since v6.4.0 the meanings of `estRows` and `actRows` for those `IndexJoin` and `Apply` operators are consistent.
```

Reply:
The last sentence is inaccurate. It's not "IndexJoin and Apply" but "the probe side children of IndexJoin and Apply".

So I wonder if "those operators" is enough. Or we should add "the probe side children of".